### PR TITLE
Issue #10115: Updated inputs in SuppressWarningsCheckTest

### LINKED ISSUE #9521: update SuppressWarningsCheckTest to use unique Input file in each test method
SuppressWarningsCheckTest.java file reuse some Input files few time in few different test method.
Expected: each test method in Test class use it own Input file and do not share it with any other test method.

more details - https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-create-Input-files-for-tests

### LINKED ISSUE #10115: Update inputs in SuppressWarningsCheckTest
Child of #10080

### PR DESCRIPTION
**Pull request by shashwatj07**
Resolves #9521

Resolves #10115

### CODE REVIEW

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsCompactNonConstant1.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
+//non-compiled with eclipse: The value for annotation attribute must be a constant expression
+package com.puppycrawl.tools.checkstyle.checks.annotation.suppresswarnings;
+import java.lang.annotation.Documented;
+
+@SuppressWarnings({"unchecked", "unused"})
+public class InputSuppressWarningsCompactNonConstant1
```
Comment (timurt) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsCompactNonConstant1.java:6]: We have to have either top comment or `Config` comment

Comment (timurt) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsCompactNonConstant1.java:6]: btw why CI does not fail with violation? no top and no config

Comment (romani) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsCompactNonConstant1.java:6]: Here is root of problem:
```
     <module name="Regexp">
       <property name="id" value="ConfigCommentOnTopInputs"/>
-      <property name="format" value="(/* Config|\A/*)"/>
+      <property name="format" value="( \* Config|/\* Config|\A/\*)"/>

```

@shashwatj07 , please create new PR to update this Regexp and place in suppression all missed violations.
after merge of it, all other PRs should be rebased.

Comment (shashwatj07) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsCompactNonConstant1.java:6]: > @shashwatj07 , please create new PR to update this Regexp and place in suppression all missed violations.
after merge of it, all other PRs should be rebased.

PR opened https://github.com/checkstyle/checkstyle/pull/10154

Comment (timurt) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsCompactNonConstant1.java:6]: merged, please rebase and check

Comment (shashwatj07) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsCompactNonConstant1.java:16]: added another commit for #10115

**Pull request merged**